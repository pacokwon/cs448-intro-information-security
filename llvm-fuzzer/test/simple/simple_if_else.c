<<<<<<< HEAD
int main()
{
=======
#include <stdio.h>

int main()
{
    char input[65536];
    fgets(input, sizeof(input), stdin);
>>>>>>> fuzzer/master
    int x = 0;

    if(x >= 0)
    {
        int y = 20 / x;
    }
    else
    {
        int z = 30 / x;
    }
}
