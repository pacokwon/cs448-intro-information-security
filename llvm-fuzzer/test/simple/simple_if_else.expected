Divide-by-zero detected at line 7 and col 20
